includes/functions.php

```diff
@@ -596,7 +596,7 @@ function yourls_get_stats( $filter = 'top', $limit = 10, $start = 0 ) {
 	if ( $limit > 0 ) {
 
 		$table_url = YOURLS_DB_TABLE_URL;
-		$results = $ydb->fetchObjects( "SELECT * FROM `$table_url` WHERE 1=1 ORDER BY `$sort_by` $sort_order LIMIT $start, $limit;" );
+		$results = $ydb->fetchObjects( "SELECT * FROM `$table_url` WHERE 1=1 ORDER BY $sort_by $sort_order LIMIT $start, $limit;" );
```

Comment:
What is the idea behind `WHERE 1=1`?

Reply:
It would make sense if this query also had a `$where_filter` (or whatever) value interpolated into it that might or might not contain additional conditions. Having it anyway is harmless (essentially zero effect on query execution time), but doesn't appear to serve any purpose in this particular case.

Reply:
It has always been there ([see in 2009...](https://github.com/YOURLS/YOURLS/blob/1.2/includes/functions.php#L357)).

I guess it's either because at this time I couldn't dare writing an SQL clause without a WHERE, or because at this time there were supposed to be potential troubles omitting this WHERE, either with MySQL or other DB engines.

I think it's also for readability to clearly specify this query DOESNT have a WHERE clause